test/0_staking.js

```diff
@@ -88,4 +90,16 @@ describe('Candidates make stakes on themselves', () => {
             expect(fStakeBN, `Stake on candidate ${candidate.staking} didn't increase`).to.be.bignumber.equal(iStakeBN.add(stakeBN));
         }
     });
+
+    it('Candidates are in validator set in the new staking epoch', async() => {
+        console.log('***** Wait for staking epoch to change');
+        await waitForValidatorSetChange(web3);
+        let validators = await waitForValidatorSetChange(web3);
+        for (candidate of constants.CANDIDATES) {
+            let validatorIndex = validators.indexOf(candidate.mining);
```

Comment:
I think there is a letter case case mismatch. `validators` are given in mixed case and `CANDIDATES` are lowercase.

Reply:
Yes, that was the problem